src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -886,14 +886,12 @@ private List<string> TryResolveFilePath(string filePath)
         /// <returns></returns>
         private string ManifestFragment(string key, string resourceString, string value, StreamWriter streamWriter)
         {
-            return string.Format(CultureInfo.InvariantCulture, "{0}# {1}{2}{0}{3:19} = {4}{2}{2}",
-                _indent, resourceString, streamWriter.NewLine, key, value);
+            return string.Create(CultureInfo.InvariantCulture, $"{_indent}# {resourceString}{streamWriter.NewLine}{_indent}{key:19} = {value}{streamWriter.NewLine}{streamWriter.NewLine}");
         }
 
         private string ManifestFragmentForNonSpecifiedManifestMember(string key, string resourceString, string value, StreamWriter streamWriter)
         {
-            return string.Format(CultureInfo.InvariantCulture, "{0}# {1}{2}{0}# {3:19} = {4}{2}{2}",
-                _indent, resourceString, streamWriter.NewLine, key, value);
```

Comment:
What could we use to replace `streamWriter.NewLine`?

Reply:
[NewModuleManifestCommand.cs](https://github.com/PowerShell/PowerShell/pull/18982/files/fb9e63251918426644920941218894b2c7dbed70#diff-4af74ffbc208d7a2ebb48c410311f8519d52cb7b9afd7b81bd9ae9f49e72f03e) is the problem (after we merge #18996 and 18997) we can close this PR